base/math.jl

```diff
@@ -867,29 +867,35 @@ end
     z
 end
 @inline function ^(x::Float32, y::Float32)
-    z = ccall("llvm.pow.f32", llvmcall, Float32, (Float32, Float32), x, y)
+    z = Float32(exp2_fast(log2(Float64(x))*y))
```

Comment:
why don't we want to trust llvm (aka libm) here anymore?

Reply:
1.6 introduced about a 3x regression on this due to a switch in which libm got loaded.

Reply:
Should we fix openlibm?

Reply:
probably. That said, I generally think that we shouldn't be relying on external algorithms, so I'm not that sad about replacing this anyway.